### PR TITLE
All spanish contents for Nostr.how | Phase 1

### PR DESCRIPTION
**Pull request by maemultichain**
Hi! This PR is attending all the issues related with SP contents. Ignore the rename parts, I corrected it because I thought these were going to be written also in Spanish but then I saw it just in English.

Everything seems to be good! Thanks.

### CODE REVIEW

File: src/routes/pages/es/guides/Astral.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Empezar con Astral
```
Comment (erskingardner) [inline src/routes/pages/es/guides/Astral.md:2]: @maemultichain can you change the filename here to be lowercase? `astral.md`

**Review by erskingardner — COMMENTED** [review]
One small comment. Do you want me to merge this as soon as the filename is changed? Or is there anything else you'd like to do before I merge.

**Comment by erskingardner** [comment]
@maemultichain you can also go ahead and add the lines required to add Spanish to the language switcher dropdown and enable routing for spanish pages That way we can deploy to a preview URL and check that everything looks good before merging to main. 

There are two things you need to change: 
1. Add a register call in `src/lib/i18n.ts` (like line 5) for `es`. 
2. Add a line to the `locales` const in `l10n.ts` (like line 9) for `es`.

**Comment by maemultichain** [comment]
Hi @erskingardner gonna fix it.

**Comment by maemultichain** [comment]
I think everything is ready, hmm the only thing I didn't see is the locales > l10n.ts. Idk if that is new but isn't shown inside my fork.
I added the es.json instead, as u can see.

**Review by erskingardner — APPROVED** [review]

**Comment by erskingardner** [comment]
@maemultichain just merged this but have found a few issues so I'll keep it off the site until I get a chance to figure out what's going on a bit later.

**Comment by maemultichain** [comment]
Sure! Anything else please let me know. Thanks for merge and the opportunity.

**Comment by maemultichain** [comment]
Sure! Anything else please let me know. Thanks for merge and the opportunity.

**Comment by maemultichain** [comment]
This is the 1 phase, we can improve if people see typos or something new to add!

**Pull request merged**